### PR TITLE
Add html link to MAV_CMD_NAV_WAYPOINT

### PR DESCRIPTION
**Pull request by Geoffroy-Milton-Innovation**

### CODE REVIEW

File: dev/source/docs/code-overview-adding-a-new-mavlink-message.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ fit in with the existing `MAVLink messages <https://mavlink.io/en/>`__.
 For example, you might want to send a new navigation command to the
 vehicle so that it can perform a trick (like a flip) in the middle of a
 mission (i.e. in AUTO mode).  In this case, you would need a new
-MAV_CMD_NAV_TRICK similar to the MAV_CMD_NAV_WAYPOINT definition
+MAV_CMD_NAV_TRICK similar to the `MAV_CMD_NAV_WAYPOINT <https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_WAYPOINT>` definition
```
Comment (Hwurzburg) [inline dev/source/docs/code-overview-adding-a-new-mavlink-message.rst:32]: as written link is not active:
```suggestion
MAV_CMD_NAV_TRICK similar to the `MAV_CMD_NAV_WAYPOINT <https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_WAYPOINT>`__  definition
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by Hwurzburg** [comment]
@Geoffroy-Milton-Innovation  please squash and I will merge...thanks

**Comment by Geoffroy-Milton-Innovation** [comment]
I didnt know how to squash and when i tried it didnt works, so I ended up deleting the repo XD

**Comment by Hwurzburg** [comment]
squashed and merged on another PR...thanks

**Pull request closed without merging**